test/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks.csproj

```diff
@@ -4,6 +4,7 @@
     <OutputType>Exe</OutputType>
     <LangVersion>7.2</LangVersion>
     <StartupObject>Microsoft.ML.Benchmarks.Program</StartupObject>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
Now that [https://github.com/dotnet/BenchmarkDotNet/issues/955](https://github.com/dotnet/BenchmarkDotNet/issues/955) has been fixed, can this be removed? I would like to not clutter up the .csproj unnecessarily.

Reply:
I am going to remove it as soon as we ship the fix to NuGet (today or tomorrow)

Reply:
solved